test/schemas/trace_validation_test.rb

```diff
@@ -278,7 +278,19 @@ def test_greater_than_failure_lesser
     assert_equal(validator.errors.first, "Element 'smaller':'1' was expected to be greater than or equal to 'larger':'5'")
   end
 
+  def test_not_validate_timestamp
+    Maze.config.span_timestamp_validation = false
+    validator = Maze::Schemas::TraceValidator.new(create_basic_request({
+      'timestamp' => '12345000000000'
+    }))
+    Time.expects(:now).never
+    validator.validate_timestamp('timestamp', Maze::Schemas::HOUR_TOLERANCE)
+    assert_nil(validator.success)
+    assert(validator.errors.empty?)
+  end
+
   def test_validate_timestamp_success_exact
+    Maze.config.span_timestamp_validation = true
```

Comment:
I think we can avoid having this in every unit test by setting it to true in the `initialize` method of `Maze::configuration`.  Then I think it will be more consistent with things like `enforce_bugsnag_integrity`.